src/components/manage/Blocks/Block/Edit.jsx

```diff
@@ -1,46 +1,35 @@
-/**
- * Edit block.
- * @module components/manage/Blocks/Block/Edit
+/*
+ * Customized copy of Edit block. It moves the block utilities outside of it
```

Comment:
@tiberiuichim @avoinea I think you removed it, but you didn't bring it back :P that's why the tests are failing.

Reply:
:thinking: Hmmm